gluoncv/data/pascal_voc/detection.py

```diff
@@ -40,14 +41,16 @@ class VOCDetection(VisionDataset):
         initialization. It often accelerate speed but require more memory
         usage. Typical preloaded labels took tens of MB. You only need to disable it
         when your dataset is extremely large.
+    generate_classes : bool, default False
```

Comment:
this is a bit confusing, we can simply use `classes=None` for default classes, and list of str to represent customized labels.
Anyway, what's the point to have custom classes for VOC?

Reply:
Okay, I got it, you want to allow custom datasets. I think using subclass is a better practice here to avoid touch VOC stuff and make it more readable.

Reply:
will use customVOC